examples/simple.js

```diff
@@ -11,12 +10,6 @@ var map = new ol.Map({
       source: new ol.source.OSM()
     })
   ],
-  controls: ol.control.defaults({
-    attributionOptions: /** @type {olx.control.AttributionOptions} */ ({
-      collapsible: false
-    })
-  }),
```

Comment:
When working on attributions, I wanted to have them collapsed by default, but there was some resistance… see https://github.com/openlayers/ol3/pull/2159